WebKitDev/Functions/Install-Xampp.ps1

```diff
@@ -16,16 +16,24 @@
   The version of XAMPP to install.
 
   .Example
-    # Install 7.2.4
-    Install-Xampp -Version 7.2.4
+    # Install 7.2.4.0
+    Install-Xampp -Version 7.2.4.0
 #>
 Function Install-Xampp {
     Param(
         [Parameter(Mandatory)]
         [string] $version
     )
 
-    $url = ('https://www.apachefriends.org/xampp-files/{0}/xampp-windows-x64-{0}-0-VC15-installer.exe' -f $version);
+    $major, $minor, $patch, $build = $version.split('.')
```

Comment:
Should we default build to 0 if not present (in case anyone else is using this)?